frontend/lib/DataModels/group_data.dart

```diff
@@ -0,0 +1,20 @@
+import 'package:adpv_frontend/DataModels/endpoint.dart';
+import 'package:adpv_frontend/DataModels/user_summary.dart';
+
+class GroupData {
+  final int id;
+  final String name;
+  List<UserSummary>? users;
+  List<Endpoint>? groupEndpoints;
```

Comment:
Myślę, że w ramach kontekstu ekranu którym się zajmujesz, te dwie zmienne (`users`, oraz `groupEndpoints`) są niepotrzebne. Przy widoku `Member'ów` doda się osobny `DataModel`. Ten nazwałbym nawet `ShortGroup`, albo coś czymś zbliżonym

Reply:
a, to wtedy to jest groupSummaty, to tego uzyje